### DISCUSSION TITLE
TriggerSmartContract

### DISCUSSION THREAD

**Discussion by zzc1131554513** [consenlabs/webview#13] [Q&A] [answered]
![c4f728add58a90bf5c073e68de2c2f8](https://user-images.githubusercontent.com/27850967/128305480-720a1953-f129-46b2-bdbf-593956de2d51.png)
![c519b1cb5d309602ef1d03f99efd540](https://user-images.githubusercontent.com/27850967/128305523-33c19062-f4cd-48e2-b30c-be4463558abd.png)

this.tronWeb = window.tronWeb;
this.myContract =  this.tronWeb.contract(usdrAbi,usdtAddr);
this.myContract.transfer(toAdr,num).send({from: formAdr})
在有些机型上  支付详情 弹窗是正确的
有些机型却是TriggerSmartContract   请问是为什么,该如何解决呢.

**Comment by xwartz** [comment, marked as answer]
应该和机型没有关系，确认一下 App 版本是否一致，在 2.9.4 版本中支持了 tron 标准转账、授权的解析

**Reply by zzc1131554513** [reply]
谢啦!确实版本问题
